tools/distpartitioning/utils.py

```diff
@@ -1,18 +1,19 @@
 import json
 import logging
 import os
+from contextlib import contextmanager
 
-import dgl
+import constants
 import numpy as np
 import psutil
 import pyarrow
 import torch
+from numpy.lib.format import open_memmap
```

Comment:
why import this here? Is it used directly in this file?

Reply:
Deleted